server/index.js

```diff
@@ -39,8 +52,37 @@ app.post('/api/user', upload.fields([
 
 const port = secrets.PORT || process.env.PORT || 3000;
 
-const server = app.listen(port, () => {
-  console.log(`listening on port ${port}...`);
-});
 
+
+// Key and Cert should be included as config for httpsServer when created.
+
+
+//NOT NEEDED
+// const httpPort = 3000;
+// const httpServer = http.createServer(app);
+// httpServer.listen(port, () => {
+//   console.log(`HTTP Server running on port ${port}`);
+// });
+
+let server = {};
+
+console.log(process.env.NODE_ENV)
```

Comment:
This may not work the way we want it to if we're relying on env variables and not secrets.
We may just be able to export one of these...maybe even the httpsServer.listen and see if the tests pass